ChromeExtension/js/background.js

```diff
@@ -9,6 +9,18 @@ function getCurrentTab() {
     });
 }
 
+function goTo(page) {
+    if (localStorage["defaultGroup"] != null && localStorage != "") {

```

Comment:
This function should only do the action i would put the if outside where you are calling from (line 92 in this file).